src/components/Login/SignUp.js

```diff
@@ -5,22 +5,23 @@ import { useHistory } from 'react-router-dom';
 import { Steps, Select, Form, Button, Input, notification } from 'antd';
 import wallet from 'wallet-besu';
 import { Main } from '@aragon/ui';
-import { createUser } from '../../lib/contracts/MasterContract';
+import { Web3Contract } from '../../utils/web3-contracts';
+import { createUser, userMap } from '../../lib/contracts/MasterContract';
 import 'antd/dist/antd.css';
 import './SingnUp.Style.css';
 
 const { Step } = Steps;
 const { Option, OptGroup } = Select;
 
-const accounts = require('../../wallet/keys.js');
+const Accounts = require('../../wallet/keys.js');
```

Comment:
Are we using this `Accounts` now since we are creating the user wallet during the signup?

Reply:
No, I changed it cause there was a small variable conflict.